jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -286,8 +286,8 @@ def test_parse_jwst_nircam_level2(self, imviz_helper):
         assert_quantity_allclose(tbl[0]['aperture_sum_counts'], 132061.576643 * u.count, rtol=1e-6)
         assert_quantity_allclose(tbl[0]['aperture_sum_counts_err'], 363.402775 * u.count)
         assert_quantity_allclose(tbl[0]['counts_fac'], 0.0036385915646798953 * (data_unit / u.ct))
-        assert_quantity_allclose(tbl[0]['aperture_sum_mag'], -6.704274 * u.mag)
-        assert_quantity_allclose(tbl[0]['flux_scaling'], 1 * data_unit)
+        assert_quantity_allclose(tbl[0]['aperture_sum_mag'], -12.804208824311152 * u.mag)
```

Comment:
Ori, good catch. Should have been this number in the test. Something isn't quite right.
```suggestion
        assert_quantity_allclose(tbl[0]['aperture_sum_mag'], 19.770234 * u.mag)
```